### ISSUE TITLE
Crash ReportAdvancedReportsViaWebCollectionViewCell.AwakeFromNib 

### ISSUE THREAD

**Issue by simonrozsival** [toggl/mobileapp#7655]
# 💥 Crash Report
The app crashes after reports are loaded.

### 👥 Impact
This has happened 80 times to 34 users so far, but we're still at 5% of the rollout.

### 🛰 Link
https://appcenter.ms/orgs/Toggl/apps/Toggl-iOS/crashes/errors/254595553u/overview

### 📝  Exception & Stack Trace
`SIGABRT: Object reference not set to an instance of an object`
```cs
    ReportAdvancedReportsViaWebCollectionViewCell.AwakeFromNib ()
    (wrapper managed-to-native) ObjCRuntime.Messaging.objc_msgSend(intptr,intptr,intptr,intptr)
    UICollectionView.DequeueReusableCell (Foundation.NSString reuseIdentifier, Foundation.NSIndexPath indexPath)
    UICollectionView.DequeueReusableCell (System.String reuseIdentifier, Foundation.NSIndexPath indexPath)
    ReportsCollectionViewSource.GetCell (UIKit.UICollectionView collectionView, Foundation.NSIndexPath indexPath)
    (wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain(int,string[],intptr,intptr)
    UIApplication.Main (System.String[] args, System.IntPtr principal, System.IntPtr delegate)
    UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName)
    Application.Main (System.String[] args)
```

I believe it's this line of code that's responsible:
```cs
TitleLabel.Text = Resources.AdvancedReportsViaWeb.ToUpper();
```
- we did not finish #7634 and the resources file doesn't contain the entries for the new strings even with the temporary English localization. It's supported by the fact that the all the reports are from users with their language set to Japanes, Portugese, or French. If I'm correct, I suppose this would cause a crash on Android as well.

**Comment by simonrozsival** [comment]
... on the other hand, I can't reproduce the issue with 🇯🇵 🤷‍♂️ 
![image](https://user-images.githubusercontent.com/374616/87927047-e1bc6180-ca82-11ea-8e03-c164a1600069.png)

**Comment by pdrj** [comment]
That is weird and very similar to the crash we had in 2.16 on Android. https://appcenter.ms/orgs/Toggl/apps/Toggl-Android/crashes/errors/354678555u/overview Looks like it might be a xamarin bug where it doesn't fallback to the base resource for some reason 🤷‍♂️

**Comment by simonrozsival** [comment]
That looks similar indeed. At least the Android crash happened just a few times, no the crashes seem to be more substantial 🤔

**Comment by simonrozsival** [comment]
What we could try is to simply copy&paste all strings to the JP/PT/FR Resources file with the English text. This way, it would behave the same way as we expect it to (show the English text), but it shouldn't crash.

**Comment by pdrj** [comment]
Yes, that's what we did on Android with that one string.

**Comment by simonrozsival** [comment]
The fix was merged into the release branch for the iOS 2.17.1 hotfix and it was also cherry-picked into the Android 2.17 release.
